torchchat/model.py

```diff
@@ -108,6 +251,8 @@ def get_recipe(cls, model_type):
             return cls._flamingo()
         elif model_type == ModelType.Llama3_1:
             return cls._llama3_1()
+        elif model_type == ModelType.Llava:
+            return cls._llava()
```

Comment:
``` python
match model_type:
    case ModelType.TextOnly:
        return cls._text_only()
    case ModelType.Flamingo:
        return cls.flamingo()
...
```

Reply:
rn it is a good timing to switch to match case statement